tcms/testcases/migrations/0010_remove_bug.py

```diff
@@ -1,20 +1,57 @@
+import glob
+import json
+
 from django.db import migrations
 
 
 def forward_copy_data(apps, schema_editor):
     bug_model = apps.get_model('testcases', 'Bug')
     link_reference_model = apps.get_model('linkreference', 'LinkReference')
 
-    for bug in bug_model.objects.all():
-        if not bug.case_run_id:
-            continue
+    link_reference_ids_file_name = '/tmp/kiwitcms-testcases-migrations-0010\
+-new-LinkReference-IDs'  # nosec:B108:hardcoded_tmp_directory
+    with open(link_reference_ids_file_name, 'w') as link_reference_ids_file:
+        for bug in bug_model.objects.all():
+            file_name = '/tmp/kiwitcms-tescases-migrations-\
+0010-Bug-%d' % bug.pk  # nosec:B108:hardcoded_tmp_directory
+            with open(file_name, 'w') as outfile:
+                json.dump(bug.serialize(), outfile)
+
+            if not bug.case_run_id:
+                continue
+
+            link_reference = link_reference_model.objects.create(
+                execution_id=bug.case_run_id,
+                name="%s %s" % (bug.bug_system.name, bug.bug_id),
+                url=bug.bug_system.url_reg_exp % bug.bug_id,
+                is_defect=True,
+            )
+
+            link_reference_ids_file.write('{},'.format(link_reference.id))
```

Comment:
Keep these IDs in a list and then json.dumps the list at the end - 1 operation for all of the ids.

Reply:
Also use `.pk` for consistency with other parts of the code.

Reply:
Do we need to format the list into json?

Reply:
json.dumps will take care of whatever it needs, you should not do any manual formatting at all.